agent/src/main/scala/io/scalac/mesmer/agent/akka/dispatcher/ActorCellConstructorAdvice.scala

```diff
@@ -0,0 +1,49 @@
+package io.scalac.mesmer.agent.akka.dispatcher
+
+import akka.actor.typed.scaladsl.adapter.ClassicActorSystemOps
+import akka.dispatch.Dispatcher
+import akka.dispatch.MessageDispatcher
+import net.bytebuddy.asm.Advice.Argument
+import net.bytebuddy.asm.Advice.OnMethodEnter
+
+import io.scalac.mesmer.core.event.DispatcherEvent.ExecutorConfigEvent
+import io.scalac.mesmer.core.event.EventBus
+
+object ActorCellConstructorAdvice {
+
+  @OnMethodEnter
+  def constructor(
+    @Argument(0) system: akka.actor.ExtendedActorSystem,
+    @Argument(3) dispatcher: MessageDispatcher
+  ): Unit =
```

Comment:
Imo this is a NOGO - we'd generate a lot of events (as this executed in the constructor of `akka.actor.ActorCell`) for single execution context (most of the actors share the ec - there is not point to generate this event for every actor)